dev_tools/scripts/add_license.sh

```diff
@@ -41,9 +40,15 @@ if ! command -v cut &> /dev/null; then
     echo 'The command `cut` is required for this script to work, but not supported by your system.' 1>&2
     exit 1
 fi
+if ! command -v date &> /dev/null; then
+    echo 'The command `date` is required for this script to work, but not supported by your system.' 1>&2
+    exit 1
+fi
 
-
-echo -e "/*${GINKGO_LICENSE_BEACON}\n$(cat ${LICENSE_FILE})\n${GINKGO_LICENSE_BEACON}*/\n" > "${COMMENTED_LICENSE_FILE}"
+CURRENT_YEAR=$(date +%Y)
+echo "${GINKGO_LICENSE_BEGIN} 2017-${CURRENT_YEAR} The Ginkgo authors" > "${COMMENTED_LICENSE_FILE}"
```

Comment:
some files use `the` Ginkgo authors, could you also change one of them? the or The

Reply:
I'm now using always `The Ginkgo authors`, since `the` was only used in two files.